src/deprecation.h

```diff
@@ -27,6 +27,8 @@ static const std::set<std::string> DEFAULT_ALLOW_DEPRECATED{{
 
     // Wallet-level features
 #ifdef ENABLE_WALLET
+    ,"z_getbalance"
+    ,"z_gettotalbalance"
```

Comment:
I hate this syntax, but see why it's been done, and we'll be re-doing this change before `gbd_oldhashes` is disabled by default, so this is fine as-is.

Reply:
The alternative is to put this block before `gbd_oldhashes` (and maybe make the same change in the order of node/wallet deprecation blocks below?) That would be resilient to future changes and wouldn't require the leading commas.

Reply:
It's only resilient while there are non-wallet features being deprecated. If we did that reorder, then the moment `gbd_oldhashes` is disabled by default, the line for the last of whatever wallet features were still enabled-by-default would need to be altered. This is just going to be a continual problem as long as the deprecations are encoded in C / C++ where we can't have trailing commas.

Reply:
True. At least that way, though, all that would be required would be the removal of a single trailing comma. But we'll fix that the next time we touch this file.

Reply:
I think C++11 onward *does* allow trailing commas in initializer lists ([first sentence of the abstract here](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2020/p2067r0.pdf)).

Reply:
Building now to check whether that works in clang.

Reply:
It does.
```suggestion
    "z_getbalance",
    "z_gettotalbalance",
```
and a comma after `"gbt_oldhashes"` above (which is just outside the suggestion range).